ruby/ql/lib/codeql/ruby/ast/Call.qll

```diff
@@ -116,7 +125,11 @@ class MethodCall extends Call instanceof MethodCallImpl {
    */
   final predicate isSafeNavigation() { super.isSafeNavigationImpl() }
 
-  override string toString() { result = "call to " + this.getMethodName() }
+  override string toString() {
+    if this instanceof SuperCall
```

Comment:
Wouldn't it be better to add a `toString` method to the `SuperCall` class instead?